Ukulele/code.py

```diff
@@ -71,56 +75,520 @@
-    23, 32, 70, 81, 22, 33, 69, 82, 21, 34, 68, 83, 20, 35, 67, 84, 19, 36, 66,
-    85, 18, 37, 65, 38, 86, 17, 64, 39, 87, 16, 63, 40, 88, 15, 62, 41, 89, 14,
-    61, 42, 90, 13, 60, 43, 91, 12, 59, 44, 92, 11, 58, 45, 93, 10, 57, 46, 94,
-    9, 56, 47, 95, 8, 55, 48, 96, 7, 54, 49, 97, 6, 53, 50, 98, 5, 52, 51, 99,
-    4, 100, 3, 101, 2, 102, 1, 103, 0,
-    ], individual_pixels=True)
-
-#Top down along both sides at once
-pixel_map_sweep = PixelMap(pixels, [
-    51, 52, 50, 53, 49, 54, 48, 55, 47, 56, 46, 57, 45, 58, 44, 59, 43, 60, 42, 61,
-    41, 62, 40, 63, 39, 64, 38, 65, 37, 66, 36, 67, 35, 68, 34, 69, 33, 70, 32, 71,
-    31, 72, 30, 73, 29, 74, 28, 75, 27, 76, 27, 77, 26, 78, 25, 79, 24, 80, 23, 81,
-    22, 82, 21, 83, 20, 84, 19, 85, 18, 86, 17, 87, 16, 88, 15, 89, 14, 90, 13, 91,
-    12, 92, 11, 93, 10, 94, 9, 95, 8, 96, 7, 97, 6, 98, 5, 99, 4, 100, 3, 101, 2, 102, 1, 103, 0
-    ], individual_pixels=True)
-
-#Every other pixel, starting at the bottom and going upwards along both sides
-pixel_map_skip = PixelMap(pixels, [
-    0, 103, 2, 101, 4, 99, 6, 97, 8, 95, 10, 93, 12, 91, 14, 89, 16, 87, 18, 85, 20,
-    83, 22, 81, 24, 79, 26, 77, 29, 74, 31, 72, 33, 70, 35, 68, 37, 66, 39, 64, 41,
-    62, 43, 60, 45, 58, 47, 56, 49, 54, 51, 52,
-    ], individual_pixels=True)
+# PIXEL MAPS: Used for reordering pixels so the animations can run in different configurations.
+# My LED strips inside the neck are accidentally swapped left-right,
+# so these maps also correct for that
+
+
+# Bottom up along both sides at once
+pixel_map_reverse = PixelMap(
+    pixels,
+    [
+        0,
+        103,
+        1,
+        102,
+        2,
+        101,
+        3,
+        100,
+        4,
+        99,
+        5,
+        98,
+        6,
+        97,
+        7,
+        96,
+        8,
+        95,
+        9,
+        94,
+        10,
+        93,
+        11,
+        92,
+        12,
+        91,
+        13,
+        90,
+        14,
+        89,
+        15,
+        88,
+        16,
+        87,
+        17,
+        86,
+        18,
+        85,
+        19,
+        84,
+        20,
+        83,
+        21,
+        82,
+        22,
+        81,
+        23,
+        80,
+        24,
+        79,
+        25,
+        78,
+        26,
+        77,
+        27,
+        76,
+        28,
+        75,
+        29,
+        74,
+        30,
+        73,
+        31,
+        72,
+        32,
+        71,
+        33,
+        70,
+        34,
+        69,
+        35,
+        68,
+        36,
+        67,
+        37,
+        66,
+        38,
+        65,
+        39,
+        64,
+        40,
+        63,
+        41,
+        62,
+        42,
+        61,
+        43,
+        60,
+        44,
+        59,
+        45,
+        58,
+        46,
+        57,
+        47,
+        56,
+        48,
+        55,
+        49,
+        54,
+        50,
+        53,
+        51,
+        52,
+    ],
```

Comment:
Please revert this change. Then disable Black for this section, and reenable it after the end of the section.